client/state/reader/reducer.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import lists from './lists/reducer';
+
+export default combineReducers( {
+	lists
```

Comment:
i think we might want two keys. one of list subscripions, and one of list objects? subs would just be an array of list IDs (maybe owner / slug? not sure what the best key is there)